corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -1020,11 +1066,26 @@ def test_long_chain_with_children(self):
         self.assertEqual(sync_log.case_ids_on_phone, all_ids)
 
 
+@use_sql_backend
+class ExtensionCasesSyncTokenUpdatesSQL(ExtensionCasesSyncTokenUpdates):
+    pass
```

Comment:
Added these SQL tests which were previously missing. Guessing that was an oversight in a previous commit.